impeller/typographer/text_frame.cc

```diff
@@ -67,12 +67,13 @@ Scalar TextFrame::RoundScaledFontSize(Scalar scale, Scalar point_size) {
   return std::round(scale * 100) / 100;
 }
 
-void TextFrame::CollectUniqueFontGlyphPairs(FontGlyphPair::Set& set,
+void TextFrame::CollectUniqueFontGlyphPairs(FontGlyphMap& glyph_map,
```

Comment:
It's hard to know if this is going to be faster.  There are multiple inserts happening now.